packages/ui-next/src/components/icons/state/clock.tsx

```diff
@@ -6,7 +6,7 @@ export const ClockIcon = memo(
   forwardRef<SVGSVGElement, StateIconProps>(
     ({ className, size, variant, ...props }, forwardedRef) => (
       <svg
-        viewBox="0 0 20 20"
+        viewBox="0 0 24 24"
```

Comment:
I think the actual icon here is `20 20`. I noticed when it was `24 24` the alignment was off in the session screen

Reply:
I re-exported from Figma to ensure it's 24x24